### PR TITLE
Fix to support nextLink with post calls for python sdk

### PR DESCRIPTION
**Pull request by najagasi**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [ ] new API version 
      - [ ] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [x] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet.
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [x] SDK of .NET (need service team to ensure code readiness)
      - [x] SDK of Python
      - [x] SDK of Java
      - [x] SDK of Js
      - [x] SDK of Go
      - [x] PowerShell
      - [x] CLI
      - [ ] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  -  [ ] Ensure to copy the existing version into new directory structure for first commit (including refactoring) and then push new changes including version updates in separate commits. This is required to review the changes efficiently.
  - Adding a new service

- [ ] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by erich-wang** [comment]
@najadojo , could you please fix CI failures?

File: specification/policyinsights/resource-manager/readme.md [hunk 1]
```diff
@@ -81,6 +81,21 @@ directive:
 
 ```
 
+### Tag: package-2021-04
+
+These settings apply only when `--tag=package-2021-04` is specified on the command line.
+
+``` yaml $(tag) == 'package-2021-04'
```
Comment (iscai-msft) [inline specification/policyinsights/resource-manager/readme.md:88]: I don't think you need this new tag. If you see, the input files are exactly the same as in the previous package. We only add a new package tag if there are changes in the files we want included in a package

**Comment by iscai-msft** [comment]
1. I've filed an [issue](https://github.com/Azure/azure-rest-api-specs/issues/14140) to fix the duplicate model names (which should fix the SDK Track2 Validation test). I believe this issue is out of the scope of the PR because it was broken before this PR, and this PR is just meant to solve the next link issue.
2. The azure-sdk-for-net check seems to be failing because of a permissions issue accessing azure-sdk-for-net, I think this is a issue of the CI, not an issue of this PR
3. it's breaking because it's fixing a bug, so this check can be failing

cc @00Kai0 @msyyc

**Review by msyyc — APPROVED** [review]

**Review by msyyc — APPROVED** [review]

**Review by 00Kai0 — APPROVED** [review]

**Comment by erich-wang** [comment]
@najagasi, although the PR is to fix bug, it is introducing breaking change. Please follow `Breaking Change Review Checklist` in description part to request breaking change change review.

**Comment by najagasi** [comment]
@erich-wang I ahve created a breaking change request https://msazure.visualstudio.com/One/_workitems/edit/9847802

**Comment by erich-wang** [comment]
@najagasi, there's dedicated team to review the breaking change once you follow instruction below to submit request.

>Hi @najagasi, one or multiple breaking change(s) is detected in your PR. Please check out the breaking change(s), and provide business justification in the PR comment and @ PR assignee why you must have these change(s), and how external customer impact can be mitigated. Please ensure to follow [breaking change policy](https://aka.ms/AzBreakingChangesPolicy) to request breaking change review and approval before proceeding swagger PR review. </br>**Action**: To initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).</br> If you want to know the production traffic statistic, please see [ARM Traffic statistic](https://portal.azure-devex-tools.com/api/pr/query-traffic?pr=Azure/azure-rest-api-specs/14035&days=5). </br> If you think it is false positive breaking change, please provide the reasons in the PR comment, report to Swagger Tooling Team via https://aka.ms/swaggerfeedback.

**Comment by erich-wang** [comment]
@najagasi, is this PR still valid? We'll close this PR if no update from you within 10 biz days, thanks.

**Comment by iscai-msft** [comment]
yeah @erich-wang it's still valid, she's trying to get in touch with the breaking changes people, but I don't think she has had any response yet. Do you know who we should tag in this PR to keep it moving? Thanks!

**Comment by erich-wang** [comment]
> yeah @erich-wang it's still valid, she's trying to get in touch with the breaking changes people, but I don't think she has had any response yet. Do you know who we should tag in this PR to keep it moving? Thanks!

There's weekly office hour hosted by review board, please refer to https://aka.ms/AzBreakingChangesPolicy

**Comment by iscai-msft** [comment]
@erich-wang thanks for your link, @najagasi have you had a chance to attend office hours? thank you so much again for your patience!

**Review by JeffreyRichter — APPROVED** [review]

**Review by msyyc — APPROVED** [review]

File: specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2019-10-01/policyStates.json [hunk 2]
```diff
@@ -1153,6 +1161,47 @@
           }
         }
       }
+    },
+    "/{nextLink}": {
+      "post": {
+        "operationId": "PolicyStates_NextLink",
```
Comment (weidongxu-microsoft) [inline specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2019-10-01/policyStates.json:1167]: The path probably need to be `{nextLink}` (without the `/`), as response have full URI.
`"@odata.context" : "https://management.azure.com/subscriptions/0b1f6471-1bf0-4dda-aec3-cb9272f09590/providers/Microsoft.PolicyInsights/policyStates/$metadata#latest/$entity"`

However the problem is that nextLink request should always be in GET, see https://github.com/Azure/azure-rest-api-specs/issues/15136

**Pull request merged**